addons/website/static/src/js/widgets/link_popover_widget.js

```diff
@@ -77,6 +77,7 @@ const NavbarLinkPopoverWidget = weWidgets.LinkPopoverWidget.extend({
                 });
             },
         });
+        this.popover.hide();
```

Comment:
@dmo-odoo This should fix it but I'm confused as to why it didn't break the test before this forward port... It should probably be there in 15.0 as well, unless we were relying on some difference in rendering pre-owl that made the popover disappear automatically...?

Reply:
Ok it was indeed the latter:
![image](https://user-images.githubusercontent.com/24205914/231127568-c5bd5402-c140-48ea-a59c-530c9a7aecf7.png)
(in `web/static/src/legacy/js/core/dialog.js`)